documentation-site/components/yard/config/radio.ts

```diff
@@ -0,0 +1,30 @@
+import {Radio, RadioGroup} from 'baseui/radio';
+import {PropTypes} from '../const';
+
+export default {
+  scopeConfig: {
```

Comment:
All `fooConfig` were renamed to `foo:` and TConfig type cast should be applied to the whole config object.

// just mentioning because it's not currently causing a build break / conflicts